Entwurf/Dokumentation.tex

```diff
@@ -0,0 +1,96 @@
+\documentclass[10pt,a4paper]{article}
+\usepackage[utf8]{inputenc}
+\usepackage[german]{babel}
+\usepackage{graphicx}
+\usepackage[table,x11names]{xcolor}
+
+\begin{document}
+
+Die Datenbank für die Applikation besteht aus vier Tabellen : Group, Reservation, Appointment und Booking.
+Die Struktur der Tabellen ist in Abbildung \ref{fig:ER-Diagramm} dargestellt.
+
+\begin{figure}[ht]		
+	\centering
+	\includegraphics[scale=0.3]{ER-Diagramm}
+	\caption{Datenbankentwurf}
+	\label{fig:ER-Diagramm}	
+	\end{figure}
+
+\begin{table}[h]
+\centering
+\caption{Appointment}
+	\label{tab:Appointment}   
+    \begin{tabular}{| l | l | l | l |}
+    \hline
+    \rowcolor{lightgray} Spalte & Datentyp & Beschreibung  \\ \hline
+    Date  & DATE & Datum des Termins(Primarykey).\\ \hline
```

Comment:
Eigentlich ist die ganze Information mit Datentyp und PK und so ja schon im Diagramm? Es wäre sinnvoller, eine Legende für das Diagramm zu machen, wo drinsteht was die Abkürzungen wie PK und FK bedeuten. In den Tabellen reicht es, wenn du einfach Spalte & Bedeutung machst

Reply:
Was man im Diagramm nicht sieht, ist, worauf die FKs referenzieren. Aber das kannst du ja im Text erklären

Reply:
wenn ich viewer wäre, würde ich gern so lesen, damit ich nicht hin und her scrollen. überzeug mich!

Reply:
Ok, dann lass es drin. Aber eine Legende für das Diagramm sollte trotzdem rein, die mag sowas